chef/data_bags/crowbar/migrate/crowbar/011_add_crowbar-upgrade-role.rb

```diff
@@ -0,0 +1,15 @@
+def upgrade(ta, td, a, d)
+  d['element_states'] = td['element_states']
+  d['element_order'] = td['element_order']
+  d['element_run_list_order'] = td['element_run_list_order']
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)